src/domains/idpay-app/api_product/app_io/policy_io.xml.tpl

```diff
@@ -57,15 +34,23 @@
                                     </set-header>
                                 </return-response>
                             </when>
+                            <when condition="@(!(Regex.IsMatch(((string)((IResponse)context.Variables["tokenstate"]).Body.As<JObject>(preserveContent: true)["fiscal_code"]), "^([A-Za-z]{6}[0-9lmnpqrstuvLMNPQRSTUV]{2}[abcdehlmprstABCDEHLMPRST]{1}[0-9lmnpqrstuvLMNPQRSTUV]{2}[A-Za-z]{1}[0-9lmnpqrstuvLMNPQRSTUV]{3}[A-Za-z]{1})$") | Regex.IsMatch(((string)((IResponse)context.Variables["tokenstate"]).Body.As<JObject>(preserveContent: true)["fiscal_code"]), "(^[0-9]{11})$")))">
+                                <return-response>
+                                    <set-status code="400" reason="Bad Request" />
+                                    <set-header name="Content-Type" exists-action="override">
+                                        <value>application/json</value>
+                                    </set-header>
+                                    <set-body>{
+                                        "code": "400",
+                                        "message": "Fiscal code not valid!"
```

Comment:
This tax code comes from SPID claims, how can it be not valid?

Reply:
The bug  reported to us is this https://pagopa.atlassian.net/browse/IDP-66
We have enforced the check to prevent cases where the value is invalid: Actually the use case that could lead us to this situation is onboarding user with postman

Reply:
Another use case is onboarding via issuer APIs. How do we prevent invalid tax codes in that case? Moreover, setting the control at api gateway level is questionable